src/lib.rs

```diff
@@ -121,6 +121,16 @@ pub enum PayloadTypeTag {
     ImmutableDataSacrificial,
     /// StructuredData type
     StructuredData,
+    /// MaidManager Account type
+    MaidManagerAccountTransfer,
+    /// DataManager Account type
+    DataManagerAccountTransfer,
+    /// PmidManager Account type
+    PmidManagerAccountTransfer,
+    /// VersionHandler Account type
+    VersionHandlerAccountTransfer,
+    /// DataManager persona level account type
+    DataManagerPersonaTransfer,
```

Comment:
Can this be renamed to DataManagerStatsTransfer ? 
No worries if its already agreed.